packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -194,17 +194,17 @@ export interface ContainerRuntimeMessage {
 const IdleDetectionTime = 5000;
 
 const DefaultSummaryConfiguration: ISummaryConfiguration = {
-    idleTime: IdleDetectionTime,
+    idleTime: IdleDetectionTime * 3,
```

Comment:
Are these the current values from service? They seem pretty high to me. 15 seconds for idle detection and 10 minutes for ack is a lot.

Reply:
I'm with you here! :) And having them clear in the code helps to see it!
Let's track changing these values as separate issues / PRs with clearly articulated whys and impact.
That the reason I want to move away from service provided configs - to be able to change them, and adapt to specific workloads.

Reply:
yes, that is what I got from the service - please confirm on your end :)  I tried on office.com and on the spo-df (both had the same settings). 

![image](https://user-images.githubusercontent.com/26266632/164355827-f8ae5c5f-2363-43c0-94f8-43b7d616f858.png)

Reply:
I confirmed in a sample - say same thing. Yes, 15 seconds is too large. We will fix it soon :)